pennylane/ops/qubit/qchem_ops.py

```diff
@@ -464,3 +464,146 @@ def _matrix(cls, *params):
     def adjoint(self):
         (theta,) = self.parameters
         return DoubleExcitationMinus(-theta, wires=self.wires)
+
+
+class QuantumNumberPreservingOR(Operation):
+    r"""QuantumNumberPreservingOR(varphi, wires)
+    1-parameter 4-qubit spatial orbital rotation gate. It can be decomposed into sequence of ``SingleExcitation(varphi, wires)``\
+    gates :math:`G(\varphi)` as follow:
+
+    |
+
+    .. figure:: ../../_static/qchem/quantum_number_preserving_or.png
+        :align: center
+        :width: 100%
+        :target: javascript:void(0);
+
+    |
+
+    The overall matrix representation for this gate is:
+
+    .. math:: U(\phi) = \begin{bmatrix}
```

Comment:
Some of the signs are different than those in the paper, please see my comment above.